backend/src/main/java/com/example/cms/ticket/TicketController.java

```diff
@@ -67,4 +68,24 @@ public ResponseEntity<List<Response>> getTicketResponses(Pageable pageable, @Pat
                 httpHeaders,
                 HttpStatus.OK);
     }
+
+    @PostMapping("/{ticketId}/responses")
+    public ResponseEntity<List<Response>> addResponse(
+            @PathVariable UUID ticketId,
+            @RequestBody ResponseDtoCreate responseDtoCreate
+    ) {
+        service.addResponse(ticketId, responseDtoCreate.getContent());
+
+        Ticket ticket = service.getTickets(Pageable.ofSize(1), Map.of("id_eq", ticketId.toString()))
```

Comment:
Nie lepiej dodać metodę do serwisu do pobierania odpowiedzi po id ticketa i tam dodać paginację?

Reply:
@mudziok będziesz to robił w tym PR czy w osobnym?

Reply:
Piszemy to na seminarium teraz będzie w tym branchu

Reply:
Rozwiązane w #[d96166c](https://github.com/AdrianKokot/put-reunice-platform/pull/123/commits/d96166c8e84fa3f61020ca0db63f7c540677cc38)